### PR TITLE
Feature/update dropdowns

### PR DESCRIPTION
**Pull request by raquel-ucl**
This will update the dropdowns to show the new options (variable to plot, season, time period and type of plot), and display an image given the user input. It only allows selecting one option per variable at the moment, and it only displays one plot.

### CODE REVIEW

**Comment by raquel-ucl** [comment]
Still can't make GH Pages work. I'm made an attempt at deploying in https://github.com/UCL/UKCORDEX-plot-explorer/tree/deploy/update_dropdowns but still no luck. I have run `npm run build` and put the build directory in the root folder, which should give GH an index.html to use. However it's still using the content of the README file 🤷.

**Comment by raquel-ucl** [comment]
Seems like deploying with the gh-pages module works well now when running `npm run deploy`. Might be the update of the react-bootstrap module that unblocked it.

File: src/components/FigureRow.js [hunk 1]
```diff
@@ -2,20 +2,25 @@ import React, { useState } from 'react';
...
-       <Figure>
-         <Figure.Image
-           alt="Image NOT found"
-           src={`${process.env.PUBLIC_URL}/images/rmse-decomposition_${plotvar.value}_${evalagainst.value}_${regmodel.value}_${timeperiod.value}.png`}
-         />
-       </Figure>
-     </Row>
-  );
+  if ([plotvar, season, period, plottype].some(el => el === null)) {
+    console.log("error, you didnt make enough selections");
```
Comment (tcouch) [inline src/components/FigureRow.js:9]: Is it an error? Do you need this?

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "dropdown2",
   "version": "0.1.0",
-  "homepage": "https://raquel-ucl.github.io/plot-explorer-clean/",
+  "homepage": "https://github-pages.ucl.ac.uk/UKCORDEX-plot-explorer/",
   "private": true,
   "dependencies": {
     "@testing-library/jest-dom": "^5.11.4",
```
Comment (tcouch) [inline package.json:7]: npm added `"bootstrap": "^5.1.3",` to my dependencies list when I managed to get this working. I'm guessing it should be added here as part of this PR

**Review by tcouch — CHANGES_REQUESTED** [review]
I think bootstrap needs adding to package.json, but otherwise looks good.

File: src/components/FigureRow.js (hunk 1 above)
Comment (raquel-ucl) [reply src/components/FigureRow.js:9]: Ah yes, I should remove that bit now.

File: package.json (hunk 2 above)
Comment (raquel-ucl) [reply package.json:7]: That's in the next PR https://github.com/UCL/UKCORDEX-plot-explorer/pull/12/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R10, but I can cherry-pick https://github.com/UCL/UKCORDEX-plot-explorer/commit/921b1826b233710fbc09fdc8f04b17ae5de15733#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519 into this branch.

File: src/components/OptionsRow.js [hunk 3]
```diff
@@ -4,63 +4,49 @@ import Row from 'react-bootstrap/Row';
...
 const plotvars = [
-  { value: 'snw', label: 'Surface Snow Amount' },
+  { value: 'fwd', label: 'Surface Snow Amount' },
+  { value: 'tas', label: 'Daily Near-Surface Air Temperature' },
   { value: 'tasmax', label: 'Daily Maximum Near-Surface Air Temperature' },
+  { value: 'tasmin', label: 'Daily Minimum Near-Surface Air Temperature' },
+  { value: 'tas99', label: 'Daily Near-Surface Air Temperature (Percentile 99)' },
+  { value: 'tas01', label: 'Daily Near-Surface Air Temperature (Percenrile 1)' },
+  { value: 'pr', label: 'Precipitation' },
+  { value: 'r99ptot', label: 'r99ptot' },
```
Comment (raquel-ucl) [inline src/components/OptionsRow.js:14]:
```suggestion
  { value: 'r99ptot', label: 'Total precipitation on days exceeding the 99th percentile' },
```

**Comment by raquel-ucl** [comment]
@tcouch are you happy to approve now? I need to start merging PRs or this will become a nightmare of conflicts :)

**Review by tcouch — APPROVED** [review]

**Comment by tcouch** [comment]
Yeah, looks good to me 👍

**Pull request merged**